### DISCUSSION TITLE
Disable Grid Lines for Rendering

### DISCUSSION THREAD

**Discussion by shafeef901** [isaac-sim/IsaacLab#1411] [General]
How do i disable these black grid lines that came with my latest pull after a couple of weeks? Feels a bit annoying when it crosses over robots.
![image](https://github.com/user-attachments/assets/3dfe216d-25a8-4f7f-aa79-524f19a0c096)

**Comment by StrainFlow** [comment]
Is this your latest pull of the Isaac Lab repository? If so can you share a version number?

**Reply by shafeef901** [reply]
Yep. 
Latest commit: 84b2d2d8849e86752aa5a948d5f818d8430eec5a
Version Number: 0.27.14

**Comment by StrainFlow** [comment]
Asking around a bit and I'm looking into how to control this setting when you create an environment.

**Comment by jb-01** [comment]
The following lines worked for me:
```python
import carb
carb.settings.get_settings().set("persistent/app/viewport/displayOptions", 0)
```

See the following post from the developer forum for reference: https://forums.developer.nvidia.com/t/how-to-disable-grid/223414/3
